src/overlays/actors/ovl_Bg_Zg/z_bg_zg.c

```diff
@@ -71,6 +71,13 @@ s32 func_808C0CC8(BgZg* this) {
 }
 
 void func_808C0CD4(BgZg* this, PlayState* play) {
+#if PLATFORM_N64
+    // Anti-piracy check, bars will not open if the check fails
+    if (IO_READ(0x800002E8) != 0xC86E2000) {
```

Comment:
```suggestion
    if (IO_READ(0x000002E8) != 0xC86E2000) {
```
When using `IO_READ` it's better to leave the address as a physical address

For some context on when this is written, it's written when IPL3 copies itself to RDRAM after RDRAM has been initialized. Specifically, this is an instruction from some embedded rsp code at offset 0x7F8 into IPL3 (`0xC86E2000` disassembles to `lqv $v14[0], ($3)`)

Reply:
could be cool to comment on those specifics in the code, just for completeness sake

Reply:
Thanks! I added this as a comment (essentially verbatim)